dwl.c

```diff
@@ -1676,23 +1673,17 @@ rendermon(struct wl_listener *listener, void *data)
 
 	/* Render if no XDG clients have an outstanding resize and are visible on
 	 * this monitor. */
-	/* Checking m->un_map for every client is not optimal but works */
-	wl_list_for_each(c, &clients, link) {
-		if ((c->resize && m->un_map) || (c->type == XDGShell
-				&& (c->surface.xdg->pending.geometry.width !=
-				c->surface.xdg->current.geometry.width
-				|| c->surface.xdg->pending.geometry.height !=
-				c->surface.xdg->current.geometry.height))) {
-			/* Lie */
-			wlr_surface_send_frame_done(client_surface(c), &now);
+	wl_list_for_each(c, &clients, link)
```

Comment:
Inner statement requires braces, this too.

Reply:
```suggestion
	wl_list_for_each(c, &stack, slink)
```

Reply:
I'm not sure what you mean by that. Stack has been removed.

Reply:
> Inner statement requires braces, this too.

I removed the if statement, so that isn't relevant anymore

Reply:
> I'm not sure what you mean by that. Stack has been removed.


Ups, I meant fstack

Reply:
Is there a reason as to why it has to be fstack?

Reply:
Sevz?

Reply:
> Is there a reason as to why it has to be fstack?

not really